core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -140,6 +142,10 @@ private DirectoryBackedArtifactSignerProvider createSignerProvider(
         config.getKeyCacheLimit());
   }
 
+  private void registerCheckAllowListHostHeader(final Router router) {
+    router.route().handler(new HostAllowListHandler(config.getHttpHostAllowList()));
+  }
+
```

Comment:
There might be a merge conflict here, this needs to be placed right besides `registerOpenApiSpecRoute` I think

Reply:
There is, i've resolve it now. Also had to install this as a global handler on the openapi router factory for it work.